tests/store/artifact/test_models_artifact_repo.py

```diff
@@ -155,6 +159,34 @@ def test_models_artifact_repo_add_registered_model_meta_file():
             overwrite=True,
             ensure_yaml_extension=False,
         )
+    # Calling the download_artifacts method on local FileStore will create an ./mlruns directory
+    # which is a test side effect. Clean this up.
+    mlruns_dir = "./mlruns"
+    if os.path.exists(mlruns_dir):
+        shutil.rmtree(mlruns_dir)
```

Comment:
Instead, can we set the artifact location to a temporary directory? If we hit an error before this block, `./mlruns` won't be cleaned up.